cobbler/remote.py

```diff
@@ -1354,6 +1443,22 @@ def remove_item(self, what: str, name: str, token: str, recursive: bool = True):
         :return: True if the action was successful.
         """
         self._log("remove_item (%s, recursive=%s)" % (what, recursive), name=name, token=token)
+        if token in self.transactions:
+            try:
+                obj_handle = self.get_item_handle(what, name, token)
+                obj = self.__get_object(obj_handle, token)
+            except ValueError:
+                return False
+            except CX:
+                return False
```

Comment:
This should be equivalent:

```suggestion
            except (ValueError, CX):
                return False
```